remove_win32_bit/jep.md

```diff
@@ -1 +1,75 @@
-TODO
+# JEP NNN: Removal of Windows 32-bit Support
+
+_Author_	Bruno Borges & George Adams
+_Owner_ Bruno Borges
+_Type_	Feature
+_Scope_	Implementation
+_Status_	Draft
+_Release_	23
+_Component_	hotspot / other
+_Discussion_	jdk dash dev at openjdk dot org
+_Effort_	?
+_Duration_	?
+_Reviewed by_	Magnus Ihse Bursie
+_Endorsed by_	Magnus Ihse Bursie
+_Created_	<date>
+_Updated_	<date>
+_Issue_	<issue#>
+
+Summary
+-------
+
+Following the deprecation of Windows 32-bit support in JEP 449, this proposal takes the next step by completely removing Windows 32-bit support from OpenJDK. 
+This decision acknowledges the evolving landscape of computing, where 64-bit systems have become predominant. 
+The removal of Windows 32-bit support will streamline development and focus resources on enhancing the performance and security of OpenJDK on modern architectures.
+
+Goals
+-----
+
+- To cease all testing and development efforts targeting the Windows 32-bit platform within the OpenJDK project.
+- To simplify OpenJDK's build and test infrastructure by focusing solely on Windows 64-bit support, aligning with current computing standards.
+
+Non-Goals
+---------
+
+- This JEP does not aim to remove 32-bit support for platforms other than Windows.
+- It does not intend to eliminate any 32-bit specific code paths that may exist that do not impact the Windows 32-bit platform.
```

Comment:
I don't understand really how this is saying anything different from the first item, except making it sound more complicated? Can't you replace both of them with something like:

```
- This JEP does not aim to remove or change 32-bit support for any platforms other than Windows.
```